guides/common/modules/proc_installing-packages-on-a-host.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Installing_Packages_on_a_Host_{context}"]
+= Installing Packages on a Host
+
+Use this procedure to review and install packages on a host using the {ProjectWebUI}.
+The list of packages available for installation depends on the Content View and Lifecycle Environment assigned to the host.
```

Comment:
I'd rather say that it depends on the activation key(s).

the AK depends on a CV, LE, and subscriptions.

Reply:
It's more accurate to say it depends on the host's content view and lifecycle environment.

Activation keys only affect the host at registration time. CV and LCE can change at any time after registration, therefore affecting the packages available for installation.